galaxy_ng/app/migrations/0003_inbound_repo_per_namespace.py

```diff
@@ -0,0 +1,30 @@
+from django.db import migrations
+
+def create_inbound_repo_per_namespace(apps, schema_editor):
+    AnsibleRepository = apps.get_model('ansible', 'AnsibleRepository')
+    AnsibleDistribution = apps.get_model('ansible', 'AnsibleDistribution')
+    Namespace = apps.get_model('galaxy', 'Namespace')
+
+    for namespace in Namespace.objects.all():
+        name = f'inbound-{namespace.name}'
```

Comment:
Can migrations make use of django.settings?  (I assume so but have never tried it)
If so, I wonder if it would be worthwhile to make the repo naming format a settings option? 

I pulled the synclist naming format out to a config option, mostly just as a way to define it one place. I don't expect it to change, though hard to tell what standalone users will want.

Reply:
I think defining the format in one place is good, I added it to settings like the synclist format. … I was thinking this migration might not need to last forever, as we are making the data “catch-up” to the code, I added `elidable=True` so it will be removed in any future squashing of migrations https://docs.djangoproject.com/en/3.0/ref/migration-operations/#django.db.migrations.operations.RunPython @alikins @cutwater - does that ^ sound right?

Reply:
If we are relying on the inbound-repo naming to be consistent  (ie, if there is code that assumes Namespace.objects.get(name='inbound-mynamespace')), at some point we may need a join table to make that connection more explicit and robust. But probably not something to worry about now, I hope.